CMakeLists.txt

```diff
@@ -176,3 +176,4 @@ add_subdirectory(apps/test-adafruitSTEMMA)
 add_subdirectory(apps/test-pid)
 add_subdirectory(apps/test-pwm)
 add_subdirectory(apps/test-pwmin)
+add_subdirectory(apps/test-pololu370-encoder)
```

Comment:
i think u meant 37D

Reply:
I will be looking to rename the whole class I think so i will tackle all of these at once